packages/paste-website/src/pages/layout/flex/index.mdx

```diff
@@ -59,8 +59,13 @@ export const pageQuery = graphql`
   categoryRoute={SidebarCategoryRoutes.LAYOUT}
   githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/layout/flex"
   storybookUrl="/?path=/story/layout-flex--flex-alignment-options"
-  data={props.data.allPasteLayout.edges}
-  packageStatus={props.data.allAirtable.edges}
+  packageName={props.data.allPasteLayout.edges[0].node.name}
```

Comment:
I think there is an opportunity here to use some lightweight data normalization to simplify the logic in these components. By using the data without normalization we are depending on the shape of the data extracted by the Airtable/graphQL integration, which is subject to change. Additionally, it makes these files dense to read, which can make it easier to introduce bugs. 

Normalization would allow us to contain our logic that extracts these nodes from the graphQL response in a single place. I think it also makes our component APIs more resilient against breaking changes from dependency changes (e.g. if we move away from gatsby).